drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/resources/config/README.txt

```diff
@@ -0,0 +1,3 @@
+This file is needed to preserve this directory (config/).
```

Comment:
I don't think so -- I just know that there's a weird error that shows up (but doesn't affect the test) "normally". So I googled it, found the solution and added this.